nixos/modules/misc/nixpkgs.nix

```diff
@@ -128,12 +132,14 @@ in
       description = ''
         List of overlays to use with the Nix Packages collection.
         (For details, see the Nixpkgs documentation.)  It allows
-        you to override packages globally. This is a function that
+        you to override packages globally. Each function in the list
         takes as an argument the <emphasis>original</emphasis> Nixpkgs.
         The first argument should be used for finding dependencies, and
         the second should be used for overriding recipes.
 
-        Ignored when <code>nixpkgs.pkgs</code> is set.
+        If <code>nixpkgs.pkgs</code> is set, overlays specified here
+        will be applied after the overlays that were already present
+        in <code>nixpkgs.pkgs</code>.
```

Comment:
Please note, that this will still ignore `nixpkgs.config`, as mentioned in https://github.com/NixOS/nixpkgs/pull/47684#discussion_r228699469.

Reply:
Added in the description for completeness.